lib/pbench/server/api/resources/query_apis/controllers_list.py

```diff
@@ -27,6 +28,14 @@ def __init__(self, config: PbenchServerConfig, logger: Logger):
             ),
         )
 
+    @Auth.token_auth.login_required()
+    def post(self):
+        return super().post()
+
+    @Auth.token_auth.login_required()
+    def get(self):
+        return super().get()
```

Comment:
Can you add a comment for why you have to do this?  I am assuming it is because the default `post()` and `get()` methods don't have `login_required`?

Reply:
Same comments for the other modules where this occurs.

Reply:
Fixed